tss-esapi/tests/integration_tests/abstraction_tests/nv_tests.rs

```diff
@@ -81,12 +80,15 @@ fn list() {
         .map(|(public, _)| public.nv_index())
         .any(|x| x == nv_index));
 
+    // Need to get the ESYS handle again, as it was closed by nv::list above
+    let owner_nv_index_handle = context
+        .tr_from_tpm_public(nv_index.into())
+        .unwrap_or_else(|_| owner_nv_index_handle.into());
```

Comment:
Replace `|_|` with `||`, there is no parameter in the closure if I am not wrong

Reply:
That's true for `Option`, but not for [`Result`](https://doc.rust-lang.org/std/result/enum.Result.html#method.unwrap_or_else). Don't think the tests would compile otherwise!